### PR TITLE
Reusable component for creating an IBM Cloud API key and getting watsonx project id

### PR DESCRIPTION
**Pull request by maurelya**
Fixes https://github.com/ibm-build-lab/VAD-VAR-Workshop/issues/315

Content changes:

- http://localhost:8000/en/watsonx/watsonxai/100#how-create-an-ibm-cloud-api-key
- http://localhost:8000/en/watsonx/watsonxai/100#how-to-lookup-your-watsonx-project-id

Code changes:

- Any code changes here

### CODE REVIEW

**Comment by MaxShapiro32** [comment]
It might be more beneficial to move the `how-create-an-ibm-cloud-api-key` section to a new `Resources` page that is accessed from the main sidebar since it is needed for other labs outside of watsonx.ai (ex. watsonx.gov).

Screenshot 2024-02-21 at 3 00 59 PM

**Comment by maurelya** [comment]
@MaxShapiro32 So, based on the meeting, we wait for @jbukuts to create a widget?

**Comment by MaxShapiro32** [comment]
@maurelya We can merge this so we at least have these instructions present for this lab. It can be updated later in [this issue](https://github.com/ibm-build-lab/VAD-VAR-Workshop/issues/338).

**Review by MaxShapiro32 — APPROVED** [review]

File: content/Watsonx/WatsonxAI/100.md [hunk 1]
```diff
@@ -171,4 +171,44 @@ If your lab requires importing a jupyter notebook to watsonx.ai, follow these st
 7. Once all the required fields are filled, click the **Create** button. You will then be launched into the notebook.
 
     ![import_notebook](./images/100/2-Create-Notebook.png)
-    
\ No newline at end of file
+
+
+## How create an IBM Cloud API key
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxAI/100.md:176]: `How create` -> `How to create`

**Review by MaxShapiro32 — APPROVED** [review]

**Pull request merged**